src/lib/types.ts

```diff
@@ -0,0 +1,13 @@
+import { ChangeEventHandler, MouseEventHandler } from 'react'
+
+export interface ExchangeInputProps {
```

Comment:
These interfaces are not global. They are used only in the ExchangeInput component. It's better to move them to the ExchangeInput.tsx file.